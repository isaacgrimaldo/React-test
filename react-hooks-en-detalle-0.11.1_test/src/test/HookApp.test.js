import React from 'react';
import '@testing-library/jest-dom';
import {shallow} from 'enzyme';

import {HookApp} from '../HookApp.js'

describe('Pruebas del archivo HookApp', () => {
    
    const wrapper = shallow( <HookApp/> )
    
    test('Debe de renderizar correctamente el componente', () => {
           expect(wrapper).toMatchSnapshot();
      });
});