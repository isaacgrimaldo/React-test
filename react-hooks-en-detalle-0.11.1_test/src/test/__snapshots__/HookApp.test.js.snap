// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del archivo HookApp Debe de renderizar correctamente el componente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
